src/expr/src/scalar.proto

```diff
@@ -588,7 +588,14 @@ message ProtoBinaryFunc {
         google.protobuf.Empty mod_uint16 = 169;
         google.protobuf.Empty mod_uint32 = 170;
         google.protobuf.Empty mod_uint64 = 171;
-        ProtoRangeContainsElemInner range_contains_elem = 172;
+        ProtoRangeContainsInner range_contains_elem = 172;
+        bool range_contains_range = 173;
+        google.protobuf.Empty range_overlaps = 174;
+        google.protobuf.Empty range_after = 175;
+        google.protobuf.Empty range_before = 176;
+        google.protobuf.Empty range_overleft = 177;
+        google.protobuf.Empty range_overright = 178;
+        google.protobuf.Empty range_adjacent = 179;
```

Comment:
nit: the `Arbitrary for BinaryFunc` is missing these variants (and probably some more), which means that we won't catch possible protobuf errors for those variants.